### DISCUSSION TITLE
Crazyflie beginner inquiring about motor direction issue.

### DISCUSSION THREAD

**Discussion by eurkaaaa** [bitcraze/discussions#1593] [Q&A]
We installed four identical motors, and the connections are exactly the same. Why does the drone automatically adjust to have two motors rotate clockwise and two counterclockwise during takeoff? Is this a design feature of the motors or is it controlled by the firmware code?
Thank you! I am looking forward to your reply.

**Comment by tobbeanton** [comment]
This is done by the wiring of the electronics.
![image](https://github.com/user-attachments/assets/8167c84e-9bf2-4b81-92f3-6b2ce2a953a9)
Here you can see that the polarity in the connector is switched, see pin order of P8 and P9.
